salt/orchestrate/apps/deploy.sls

```diff
@@ -1,5 +1,5 @@
 {% set env_settings = salt.cp.get_url("https://raw.githubusercontent.com/mitodl/salt-ops/main/salt/environment_settings.yml", dest=None)|load_yaml %}
-{% set ENVIRONMENT = salt.environ.get('ENVIRONMENT', 'rc-apps') %}
+{% set ENVIRONMENT = salt.environ.get('ENVIRONMENT', 'applications-qa') %}
```

Comment:
Pulumi treats the env name as applications-qa, but there are still too many assumptions built into the pillar data for us to switch to that naming in salt land.